source/module_hamilt_lcao/module_gint/kernels/cuda/cuda_tools.cu

```diff
@@ -0,0 +1,227 @@
+#include <iostream>
+
+#include "module_hamilt_lcao/module_gint/kernels/cuda/cuda_tools.cuh"
+cudaError_t checkCuda(cudaError_t result)
+{
+#if defined(__DEBUG)
+    if (result != cudaSuccess)
+    {
+        fprintf(stderr, "CUDA Runtime Error: %s\n", cudaGetErrorString(result));
+        assert(result == cudaSuccess);
+    }
```

Comment:
These checks should be also executed in non-debug environment. Same below.

Reply:
solved